src/components/SingleMovieDetails/SingleMovieDetails.js

```diff
@@ -1,10 +1,42 @@
-// function SingleMovieDetails( {singleMovieDetail} ) {
-//     return(
-//         <div id={`${movie.id}`}>
-//             <img src={movie.image} height="50%" width="50%" />
-//             <h2>{`${movie.title}`}</h2>
-//         </div>
-//     )
+import { useEffect } from "react";
+import PropTypes from 'prop-types';
+
+function SingleMovieDetails({
+ selectedMovie ,
+  setSelectedMovie,
+  singleMovieDetail,
+  setSingleMovieDetail,
+  getIndividualMovie,
+}) {
+  useEffect(() => {
+    // console.log('selectedMovie:', selectedMovie)
+      selectedMovie && getIndividualMovie(selectedMovie)
+      .then((data) => setSingleMovieDetail(data.movie))
+      .catch((error) =>
+        console.error("Error fetching individual movie:", error)
+      );
+      console.log("selectedMovieDetail:", singleMovieDetail);
```

Comment:
Rather than just console logging in the catch block, think about if there is anything you want to display to the user in the event of errors (or anything that triggers the catch block).   How might you use the catch block to help you accomplishing displaying helpful messaging to the user?